control/veda_deliver_cielo.py

```diff
@@ -117,13 +117,13 @@ def perform_transcript(self, job_id, lang_code):
         """
         Request cielo24 to generate transcripts for a video.
         """
-        callback_url = build_url(
+        callback_url = '{}?job_id={}&iwp_name={}&lang_code={}&org={}&video_id={}'.format(
```

Comment:
same as above

Reply:
This will make the line longer and pylint will complain.